### PR TITLE
Use builders for condition tests

### PR DESCRIPTION
**Pull request by dibyom**
# Changes

In #1127, we added test builders for Conditions.
Now, let's actually use them!

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: test/pipelinerun_test.go [hunk 1]
```diff
@@ -25,17 +25,19 @@ import (
...
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
-	tb "github.com/tektoncd/pipeline/test/builder"
+	"github.com/tektoncd/pipeline/pkg/artifacts"
+
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/apis"
 	knativetest "knative.dev/pkg/test"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
```
Comment (sbwsg) [inline test/pipelinerun_test.go]: the package ordering looks a little funky to me here.  looks like atm it goes:

* stdlib
* k8s.io api machinery
* pipeline/artifacts
* k8s.io other stuff
* pipeline other stuff

Maybe an IDE snafu?

Comment (dibyom) [reply test/pipelinerun_test.go]: Indeed. Ran goimports on this file manually.

**Comment by dibyom** [comment]
Import ordering should be fixed now!

**Pull request merged**